cli/cmd/deploy.go

```diff
@@ -53,21 +55,47 @@ func deploy(force bool, ignoreCache bool) {
 		errors.Exit(err)
 	}
 
-	zipInput := &zip.Input{
+	params := map[string]string{
+		"force":       s.Bool(force),
+		"ignoreCache": s.Bool(ignoreCache),
+	}
+
+	configBytes, err := ioutil.ReadFile("cortex.yaml")
+	if err != nil {
+		errors.Exit(errors.Wrap(err, "failed to read configuration file"))
+	}
+
+	projectPaths, err := files.ListDirRecursive(root, false,
+		files.IgnoreCortexYAML,
+		files.IgnoreHiddenFiles,
+		files.IgnoreHiddenFolders,
+		files.IgnorePythonGeneratedFiles,
+	)
+	if err != nil {
+		errors.Exit(err)
+	}
+
+	projectZipBytes, err := zip.ToMem(&zip.Input{
 		FileLists: []zip.FileListInput{
 			{
-				Sources:      allConfigPaths(root),
+				Sources:      projectPaths,
 				RemovePrefix: root,
 			},
 		},
+	})
+
+	if err != nil {
+		errors.Exit(errors.Wrap(err, "failed to zip configuration file"))
 	}
 
-	params := map[string]string{
-		"force":       s.Bool(force),
-		"ignoreCache": s.Bool(ignoreCache),
+	uploadInput := &HTTPUploadInput{
+		Bytes: map[string][]byte{
+			"config.yaml": configBytes,
```

Comment:
since we only have one config file now, I think it should be named "cortex.yaml" here too to avoid confusion